src/services/useFetch.js

```diff
@@ -0,0 +1,34 @@
+import React from'react';
+ 
+constuseFetch=()=>{​​​​​​​​
```

Comment:
@berni23 se debería arreglar el formato de este fichero porque está todo colapsado

Reply:
@all-contributors please add @luissastresan  for code